backend/src/middleware/appContext.js

```diff
@@ -1,6 +1,8 @@
-var { models } = require("../models");
+const models = require("../models");
 
 module.exports = async (req, res, next) => {
-  req.context = { models };
+  req.context = {
+    models: models
+  };
```

Comment:
What's the purpose of this change? This works just as well:

```suggestion
  req.context = { models };
```
This way you can require as needed, and toss it into the context directly

Reply:
As a sidenote, I'm going to make a ticket for a small task to move `req.user` to `req.context.user`. The purpose of context is to provide app context for the request. `req.user` makes it sound like it's part of the request, not the context surrounding it.

That or we can drop context, since it's pretty verbose for not much. We can discuss it.